packages/fxa-settings/src/components/ModalVerifySession/index.tsx

```diff
@@ -109,26 +112,41 @@ export const ModalVerifySession = ({
           });
         })}
       >
-        <h2
-          id="modal-verify-session-header"
-          className="font-bold text-xl text-center"
-        >
-          Verify your email
-        </h2>
-        <p
-          id="modal-verify-session-desc"
-          data-testid="modal-desc"
-          className="my-6 text-center"
+        <Localized id="mvs-verify-your-email">
+          <h2
+            id="modal-verify-session-header"
+            className="font-bold text-xl text-center"
+          >
+            Verify your email
+          </h2>
+        </Localized>
+
+        <Localized
+          id="mvs-enter-verification-code-desc"
+          vars={{ email: primaryEmail.email }}
+          elems={{
+            email: <span className="font-bold"></span>,
+          }}
         >
-          Please enter the verification code that was sent to{' '}
-          <span className="font-bold">{primaryEmail.email}</span> within 5
-          minutes.
-        </p>
+          <p
+            id="modal-verify-session-desc"
+            data-testid="modal-desc"
```

Comment:
How about `modal-verify-session-desc`, so the testid will be unique to this modal